cypress/support/components/HomePage.js

```diff
@@ -13,19 +13,14 @@ export function ClickSubmitButton()             { cy.Get('[data-testid="model-cr
 export function UploadImportModelFile(name)     { cy.UploadFile(name, `[data-testid=model-creator-import-file-picker] > div > input[type="file"]`)}
 
 export function ClickDeleteModelButton(row)     { cy.Get(`[data-list-index="${row}"] > .ms-FocusZone > .ms-DetailsRow-fields`).find('i[data-icon-name="Delete"]').Click() }
-export function ClickConfirmButton()            { cy.Get('.ms-Dialog-main').contains('Confirm').Click() }
+export function ClickConfirmButton()            { return cy.Get('.ms-Dialog-main').contains('Confirm').Click() }
```

Comment:
Does Click() really return a boolean?

Reply:
No, it does not return a Boolean. It returns the promise that allows us to chain the command with a .then() call which allows our code to execute in the proper time frame. This change was necessary and correct.